### PR TITLE
Add Prism styling to the <code> markdown blocks

### PR DESCRIPTION
**Pull request by MikeKingdom**
This adds  Prismjs markdown styling to `<code>` blocks in the blogs and platform docs.
The code styling is done via some css still but the rest of the Markdown is styled by grommet components. For now I just picked an existing dark CSS for the `<code>` until we settle on something.

Note that some of the markdown content files still don't render code blocks correctly since Grommet's Markdown component seems to expect a blank line before the **\`\`\`language**  lines and some of the checked in content doesn't put a blank line before them yet.

See /blog/2018-02-01-using-hpe-onesphere-javascript-package/ as an example of working formatting

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ module.exports = {
               wrapperStyle: 'margin-bottom: 1.0725rem',
             },
           },
-          'gatsby-remark-prismjs',
+          // 'gatsby-remark-prismjs',
```
Comment (karatechops) [inline gatsby-config.js]: It's probably safe to remove this dependency and smartypants from the `the package.json`

File: src/components/Markdown/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      return <Download color="brand" />;
+    }
+    return <GrommetImage {...this.props} />;
+  }
+}
+
+Image.propTypes = {
+  src: PropTypes.string,
+};
+
```
Comment (karatechops) [inline src/components/Markdown/index.js]: Let's document this Image component with comments, it's not quite clear how this is implemented in the markdown.

Comment (MikeKingdom) [reply src/components/Markdown/index.js]: I was thinking about actually making an Anchor that smartly adds the right icon depending on the URL so this way of using grommet icons via markdown isn't needed

File: gatsby-config.js (hunk 1 above)
Comment (MikeKingdom) [reply gatsby-config.js]: Will do in the next PR

File: src/components/Markdown/index.js (hunk 2 above)
Comment (MikeKingdom) [reply src/components/Markdown/index.js]: Will do in the next PR

**Pull request merged**